lib/main/atom/commands/goToDeclaration.ts

```diff
@@ -27,12 +38,19 @@ commands.set("typescript:go-to-declaration", deps => {
     } else {
       open(result.body![0])
     }
-
-    function open(item: {file: string, start: {line: number, offset: number}}) {
-      atom.workspace.open(item.file, {
-        initialLine: item.start.line - 1,
-        initialColumn: item.start.offset - 1
-      })
-    }
   }
-})
+});
+
+commands.set("typescript:return-from-declaration", deps => {
+   return async e => {
+      const position = prevCursorPositions.pop();
+      if (!position) {
+         atom.notifications.addInfo('AtomTS: Previous position not found.');
+         return;
+      }
+      open({
+         file: position.file,
+         start: { line: position.line, offset: position.offset }
+      });
```

Comment:
Not completely sure what you mean here. position doesn't have a `start` attribute, it looks like:

    {
        file: string,
        line: number,
        offset: number
    }

Open expect the `line` and `offset` to be children of `start`, which is why I wrote it out like that.  Are you saying to just change the definition of `open` to correspond with the `position` argument?